### ISSUE TITLE
About Layout Estimation Network Training

### ISSUE THREAD

**Issue by EralpKocas** [chengzhag/Implicit3DUnderstanding#5]
Hello, I am trying to reproduce your baseline. I prepared data and other stuff by following your instructions. When I started to train both LEN and LIEN+LDIF, I saw that the loss convergence is as expected for LIEN+LDIF but I do not think the LEN trains as expected. Even though the best model is trained in 4th epochs and remain the same until 53th epochs (I trained until 53 so far and it does not seem promising), the config file says it will be trained for 100 epochs. Thus, I would like to ask you whether any problem with LEN or what I should expect as a traning behavior? Thank you.

**Comment by chengzhag** [comment]
Hi, if the loss of LEN is something like [this](https://github.com/yinyunie/Total3DUnderstanding/issues/19#issue-719753883), it is expected. We use the LEN from Total3DUnderstanding, and we confirmed with the author that [the overfitting exists](https://github.com/yinyunie/Total3DUnderstanding/issues/19#issuecomment-709138488). Also, the best model seems to have reached the performance in Total3DUnderstanding.

**Comment by EralpKocas** [comment]
Thank you for your answer. As far as we see in our training results both training curves and visualization, it seems to be fine! Thank you for your work and your time.
